crd/nodenetworkconfig/api/v1alpha/nodenetworkconfig.go

```diff
@@ -62,10 +62,17 @@ const (
 type NodeNetworkConfigStatus struct {
 	// +kubebuilder:default=0
 	// +kubebuilder:validation:Optional
-	AssignedIPCount   int                `json:"assignedIPCount"`
-	Scaler            Scaler             `json:"scaler,omitempty"`
-	Status            Status             `json:"status,omitempty"`
-	NetworkContainers []NetworkContainer `json:"networkContainers,omitempty"`
+	AssignedIPCount          int                      `json:"assignedIPCount"`
+	Scaler                   Scaler                   `json:"scaler,omitempty"`
+	Status                   Status                   `json:"status,omitempty"`
+	NetworkContainers        []NetworkContainer       `json:"networkContainers,omitempty"`
+	NetworkContainerStatuses []NetworkContainerStatus `json:"networkContainerStatuses,omitempty"`
+}
+
+// NetworkContainerStatus defines the status of the Network Containers
+type NetworkContainerStatus struct {
```

Comment:
any reason not to have this be a field within `NetworkContainer`?

Reply:
My argument for that is the Scaler and Status fields were not included in the Network Containers as well. My thought process is to avoid breaking change, so continued along that path.

Reply:
Updated to include it into the Network container with a timestamp.